src/logger.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * A method that returns the default logger for this application
+ * Uses a function instead of a constant to allow mocking
```

Comment:
Just so this doesn't look like a run-on sentence in VS Code's popover view:

```suggestion
 * A method that returns the default logger for this application.
 * Uses a function instead of a constant to allow mocking.
```

Reply:
It's not a complete sentence though. My high school English teachers would be mortified!

Reply:
You could always make them complete sentences, lol